package.json

```diff
@@ -22,11 +22,12 @@
     "url": "https://github.com/lifeomic/app-tools/issues"
   },
   "dependencies": {
-    "client-oauth2": "^4.2.3",
+    "client-oauth2": "https://github.com/Druotic/js-client-oauth2.git#remove-redirect-pathname-match-check",
     "query-string": "^6.2.0"
   },
   "devDependencies": {
     "@types/jest": "^23.3.10",
+    "@types/node": "^13.1.2",
```

Comment:
I found that locally link would break the typings without this. Not strictly necessary, but makes local development easier.